### ISSUE TITLE
spaCy NER not working!

### ISSUE THREAD

**Issue by azarezade** [RasaHQ/rasa_nlu#1113]
I wanted to compare the accuracy of `ner_spacy` with `ner_crf`, but after training I get the following  results, which`ner_crf` works very well but `ner_spacy` returns zeros accuracy!
Moreover, `ner_spacy` also shows the entities for its default NER, which is undesirable for me.

<img width="574" alt="screen shot 1397-03-07 at 11 26 06 am" src="https://user-images.githubusercontent.com/11195749/40612787-28ca3d2e-6291-11e8-82f7-1e701aa748b7.png">

<img width="1192" alt="screen shot 1397-03-07 at 11 26 59 am" src="https://user-images.githubusercontent.com/11195749/40612792-2c20d578-6291-11e8-9c55-693462072ee5.png">

**Rasa NLU version**: 0.12.3

**Operating system** : Mac OS X

**Content of model configuration file**:
```yml
language: "fa"
pipeline:
- name: "nlp_spacy"
- name: "tokenizer_spacy"
- name: "ner_spacy"
- name: "ner_crf"
- name: "ner_synonyms"
- name: "intent_featurizer_count_vectors"
- name: "intent_classifier_tensorflow_embedding"
```

**Comment by akelad** [comment]
I'm afraid ner_spacy isn't configurable to select only certain types of entities to extract. What does your training data look like? it's unlikely that all those `ner_spacy` entities are present in your data anyways. But also, evaluating ner_spacy isn't really that useful, because you won't have labeled these entities in your training data and therefore it has no labels to compare to whether it's correctly identified the entities

**Comment by azarezade** [comment]
Yes all entities (except the default spaCy entities; far, loc, org, perf, pro) are in the training data.
But I think the training data is valid, since `ner_crf` learned the entities with avg precision of about 0.97!

**Comment by akelad** [comment]
Oh, but ner_spacy is a pretrained model only. It doesn't train on the labeled data you created

**Comment by azarezade** [comment]
Oh, I didn't know that!
Since in spaCy you can learn entity on your labeled data, so I thought that n`ner_spacy` would learn entities from my own data as well!
So is there any plan to add this feature to rasa `ner_spacy`?

**Comment by amn41** [comment]
you can train your own (or fine tune) spaCy NER model but you have to be thoughtful about datasets etc, hence why we don't do that automatically. you can check the spacy docs for how to do it.

**Comment by azarezade** [comment]
Thanks Alan, yes I know. I have actually played with [spacy NER training](https://github.com/explosion/spaCy/blob/master/examples/training/train_ner.py) some times ago. In my experience even with the default settings in [this code](https://github.com/explosion/spaCy/blob/master/examples/training/train_ner.py), the entity recognizer is good. To be included in rasa, someone just need the convert the data to the required format.

I can also report the results of spacy NER for the above dataset (when its done) to better compare with `ner_crf`.

**Comment by amn41** [comment]
that would be great! would love to see a comparison
